source/synthDrivers/_espeak.py

```diff
@@ -121,22 +121,28 @@ class espeak_VOICE(Structure):
 	def __eq__(self, other):
 		return isinstance(other, type(self)) and addressof(self) == addressof(other)
 
+# constants that can be returned by espeak_callback
+callback_continue_synthesis=0

```

Comment:
Great! You worked out what these values were for. I think they should be in caps though.